platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,346 @@
+# Threshold Signature Scheme (TSS) Ledger ID
```

Comment:
Calling this proposal `TSS Ledger Id` isn't super descriptive, as the ledger ID is only a tiny slice of what the proposal covers. What about `TSS Keying`? Or `TSS Key Distribution`?

Reply:
The naming was chosen more for the PM and business abstraction level.  This proposal delivers the `Ledger Id`.   The shares and logistics with keying are all in support of the existence and transfer of the `Ledger Id`.